src/binaries/BinariesManager.ts

```diff
@@ -8,7 +8,6 @@ import { Constants } from '../Constants';
 import { Logger } from '../logger/Logger';
```

Comment:
Following your changes, the whole BinariesManager class seems quite empty now. Please consider dropping it and calling directly BinariesUtility (so the class called "BinariesUtilityV2" once it's renamed to drop the "V2") wherever you used to call BinariesManager.

Reply:
Sure, we will be doing that in the next PR. As mentioned to keep things simple I did not do a lot of deletions but will be doing that slowly.